### PR TITLE
Initial commit of JankStats samples

### PR DESCRIPTION
**Pull request by chethaase**

### CODE REVIEW

**Comment by wojtek-kalicinski** [comment]
I'd suggest renaming the directory to JankStatsSample to match other samples in this repo, and also editing the root README file to add a link and description of JankStatsSample, similar to the other samples

**Review by wojtek-kalicinski — CHANGES_REQUESTED** [review]
I'd suggest renaming the directory to JankStatsSample to match other samples in this repo, and also editing the root README file to add a link and description of JankStatsSample, similar to the other samples

**Comment by mlykotom** [comment]
I will iterate on it later, to add e.g. bindings to various android classes

File: JankStatsSample/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Benchmark Sample
```
Comment (wojtek-kalicinski) [inline JankStatsSample/README.md:1]: s/Benchmark/JankStats/ ?

File: JankStatsSample/app/proguard-rules.pro [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```
Comment (wojtek-kalicinski) [inline JankStatsSample/app/proguard-rules.pro:1]: remove this file if everything's commented?
(and remove from build.gradle if referenced)

**Review by wojtek-kalicinski — APPROVED** [review]

**Pull request merged**